datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/logos.py

```diff
@@ -0,0 +1,91 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import click
+from PIL import Image
+
+
+from ...constants import NOT_TILES, get_root
+from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_success, echo_waiting
+from ...utils import get_valid_integrations, load_manifest
+
+
+REQUIRED_IMAGES = {
+    'avatars-bot.png': (128, 128),
+    'saas_logos-bot.png': (200, 128),
+    'saas_logos-small.png': (120, 60)
+}
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    short_help='Validate logos files'
+)
+@click.argument('check', required=False)
+def logos(check):
+
+    """Validate logo files."""
+
+    if check:
+        checks = [check]
+    else:
+        checks = sorted(get_valid_integrations())
+
+    blacklisted_integrations_msg = ''
+    count_successful = 0
+    errors = dict()
+    error_checks = set()
+
+    for check in checks:
+        display_name = load_manifest(check).get('display_name', check)
+        if check in NOT_TILES:
+            blacklisted_integrations_msg += '{} does not currently have an integration tile.\n'.format(display_name)
+            continue
+
+        path_to_check_logos = os.path.join(get_root(), check, 'logos')
+
+        for logo, required_size in REQUIRED_IMAGES.items():
+            logo_file_name = os.path.join(path_to_check_logos, logo)
+            if not os.path.isfile(logo_file_name):
+                errors[logo] = '    {} is missing for {}'.format(logo, display_name)
+            else:
+                width, height = get_resolution(logo_file_name)
```

Comment:
Can we simplify this in just `size` since we never use width and height independently ?